test/test_dataclass_validations.py

```diff
@@ -1,112 +1,115 @@
-        browser_params[0].tp_cookies = "something unsupported"
-        with pytest.raises(ConfigError):
-            validate_browser_params(browser_params[0])
-        browser_params[0].tp_cookies = "never"
-        try:
-            validate_browser_params(browser_params[0])
-        except:
-            pytest.fail("Unexpected error raised")
-
-    def test_save_content_type(self):
-        _, browser_params = self.get_config()
-        browser_params[0].save_content = []
-        with pytest.raises(ConfigError):
-            validate_browser_params(browser_params[0])
-        browser_params[0].save_content = "something unsupported"
-        with pytest.raises(ConfigError):
-            validate_browser_params(browser_params[0])
-        browser_params[0].save_content = False
-        try:
-            validate_browser_params(browser_params[0])
-        except:
-            pytest.fail("Unexpected error raised")
-        browser_params[0].save_content = "script"
-        try:
-            validate_browser_params(browser_params[0])
-        except:
-            pytest.fail("Unexpected error raised")
-
-
-class TestManagerParams(TestDataclassValidations):
-    def test_log_file_extension(self):
-        manager_params, _ = self.get_config()
-        manager_params.log_file = "something.unsupported"
-        with pytest.raises(ConfigError):
-            validate_manager_params(manager_params)
-        manager_params.log_file = []
-        with pytest.raises(ConfigError):
-            validate_manager_params(manager_params)
-
-    def test_database_file_extension(self):
-        manager_params, _ = self.get_config()
-        manager_params.database_name = "something.unsupported"
-        with pytest.raises(ConfigError):
-            validate_manager_params(manager_params)
-
-    def test_failure_limit(self):
-        manager_params, _ = self.get_config()
-        manager_params.failure_limit = "not None and not int"
-        with pytest.raises(ConfigError):
-            validate_manager_params(manager_params)
-        manager_params.failure_limit = None  # when failure_limit is set to None
-        try:
-            validate_manager_params(manager_params)
-        except:
-            pytest.fail("Unexpected error raised")
-        manager_params.failure_limit = 2  # when failure_limit is set to int
-        try:
-            validate_manager_params(manager_params)
-        except:
-            pytest.fail("Unexpected error raised")
-
-    def test_output_format(self):
-        manager_params, _ = self.get_config()
-        manager_params.output_format = "not None and not int"
-        with pytest.raises(ConfigError):
-            validate_manager_params(manager_params)
-        manager_params.output_format = "s3"
-        try:
-            validate_manager_params(manager_params)
-        except:
-            pytest.fail("Unexpected error raised")
+def get_config(data_dir=""):
+    return ManagerParams(), [BrowserParams()]
```

Comment:
This is no longer needed right?

Reply:
I am calling get_config() in every function to get default params both manager and browser